probe/probe.go

```diff
@@ -302,3 +334,362 @@ func probeNode(c *k8s.Client) {
 		fmt.Println("No kubernetes environment found")
 	}
 }
+
+type KubeArmorProbeData struct {
+	OSImage                 string
+	KernelVersion           string
+	KubeletVersion          string
+	ContainerRuntime        string
+	ActiveLSM               string
+	KernelHeaderPresent     bool
+	HostSecurity            bool
+	ContainerSecurity       bool
+	ContainerDefaultPosture tp.DefaultPosture
+	HostDefaultPosture      tp.DefaultPosture
+}
+
+func isKubeArmorRunning(c *k8s.Client) bool {
+	_, err := getKubeArmorDaemonsets(c)
+	return err == nil
+
+}
+
+func getKubeArmorDaemonsets(c *k8s.Client) (bool, error) {
+	var data [][]string
+	// // KubeArmor DaemonSet
+	w, err := c.K8sClientset.AppsV1().DaemonSets("kube-system").Get(context.Background(), "kubearmor", metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+	color.Green("\nFound KubeArmor running in Kubernetes \n\n")
```

Comment:
```suggestion
	color.Green("\nFound KubeArmor running in Kubernetes\n\n")
```
Let's not have trailing or preceding spaces if there's nothing after or before it!